.travis.yml

```diff
@@ -10,6 +11,10 @@ jdk:
 git:
   submodules: false
 
+before_install:
+  - curl -o- -L https://yarnpkg.com/install.sh | bash
```

Comment:
why we need to install yarn now?

Reply:
`yarn` version that is available on Travis is using an old version and Travis doesn't give a way to change its version.
That command installs the latest version of yarn and so that  we can remove the deprecation warning on `Buffer()`